content/background.html

```diff
@@ -58,22 +58,21 @@ <h3> Ethnicity</h3>
 
 <div class="vbx2"></div>
 
-<h3>Mother Tongue</h3>
+<h3>Sexual Orientation</h3>
 <p class="half">
   <span class="big">
-  45% of respondents at least speak two languages, with English not being their first language.
+  6.9% of respondents are part of the LGBTQ+ community.
```

Comment:
I was going through this and i think that's not a fair comparison -- if you are trans, but straight, you would still be a part of the LGBTQ+ community (and we wouldn't see that, since the person would answer that they're hetero, in that case)

would it be better to word it as "at least 6.9 %" are part of the community?